npc/cities/niflheim.txt

```diff
@@ -339,13 +343,18 @@ niflheim,350,258,1	script	Cursed Spirit#nif	4_NFWISP,{
 				close;
 			}
 		}
-		monster "niflheim",345,259,"Orc Skeleton",1462,1,"Cursed Spirit#nif::OnMyMobDead";
-		monster "niflheim",347,261,"Orc Skeleton",1462,1,"Cursed Spirit#nif::OnMyMobDead";
-		monster "niflheim",344,253,"Orc Skeleton",1462,1,"Cursed Spirit#nif::OnMyMobDead";
-		monster "niflheim",346,251,"Orc Skeleton",1462,1,"Cursed Spirit#nif::OnMyMobDead";
-		monster "niflheim",349,249,"Orc Skeleton",1462,1,"Cursed Spirit#nif::OnMyMobDead";
-		monster "niflheim",350,260,"Orc Skeleton",1462,1,"Cursed Spirit#nif::OnMyMobDead";
-		monster "niflheim",353,256,"Orc Skeleton",1462,1,"Cursed Spirit#nif::OnMyMobDead";
+		if (.last_mob_summon + 60 < gettimetick(2)) { // only spawn every 60 seconds, to prevent farming exploits
+			.@label$ = sprintf("%s::OnMyMobDead", strnpcinfo(NPC_NAME_UNIQUE));
+			killmonster("niflheim", .@label$);
+			.last_mob_summon = gettimetick(2);
+			monster("niflheim", 345, 259, "--ja--", G_ORC_SKELETON, 1, .@label$);
+			monster("niflheim", 347, 261, "--ja--", G_ORC_SKELETON, 1, .@label$);
+			monster("niflheim", 344, 253, "--ja--", G_ORC_SKELETON, 1, .@label$);
+			monster("niflheim", 346, 251, "--ja--", G_ORC_SKELETON, 1, .@label$);
+			monster("niflheim", 349, 249, "--ja--", G_ORC_SKELETON, 1, .@label$);
+			monster("niflheim", 350, 260, "--ja--", G_ORC_SKELETON, 1, .@label$);
+			monster("niflheim", 353, 256, "--ja--", G_ORC_SKELETON, 1, .@label$);
```

Comment:
Why not `areamonster()`?

Reply:
Because Gravity seemed to like encircling people with Orc Skeletons.

Reply:
For more dramatic effect.

Reply:
Haha... I like! :heartpulse: